### DISCUSSION TITLE
Problem connecting to Redis Resource

### DISCUSSION THREAD

**Discussion by 1800joe** [noop-inc/documentation#63] [Bugs]
It appears the Service container is unable to reach the Redis resource.

Using v0.2.0

<img width="908" alt="Screenshot 2024-01-09 at 4 19 36 PM" src="https://github.com/noop-inc/documentation/assets/3111670/395f9c66-d1ce-4360-8fbd-54e69fad25f9">

**Comment by arbitrarytech** [comment]
We've identified an issue with Resource network forwarding in cases with more than one Resource in the Environment. A fix is in the works. Did this Environment have multiple Resources?

**Reply by 1800joe** [reply]
Yes, it also had a postgres db

**Reply by arbitrarytech** [reply]
A fix for this was released in Desktop v0.3.0 and worked around a bug in containerd that sometimes resulted in multiple containers trying to bind to the same host port.

**Reply by arbitrarytech** [reply]
Looks like the Resource networking fix wasn't the only issue. We're aware of a TLS issue that is only effecting Redis, likely causing your issues.

**Comment by 1800joe** [comment]
Fixed as of v0.8.0
